.storybook/utils/I18next/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import i18n from "i18next";
+import { initReactI18next, useTranslation } from "react-i18next";
+import I18nProvider from "../../../src/components/I18nProvider";
+import en from "../../locales/en/app.json";
+
+const match = window.location.href.match(/locale=(.[^&]*)/);
+const locale = match ? match[1] : "en";
+
+i18n.use(initReactI18next).init({
+  resources: {
+    en: {
```

Comment:
Don't we need a `locales/en/carbon.json` added here, where are the components in storybook getting their translations from?